samples/snippets/usage_guide.py

```diff
@@ -71,6 +71,33 @@ def client_list_entries(client, to_delete):  # pylint: disable=unused-argument
         # [END client_list_entries_order_by]
         break
 
+    # [START logging_list_gke_audit_logs]
+    import google.cloud.logging
+    from datetime import datetime, timedelta, timezone
+    import os
+
+    # pull your project id from an environment variable
+    project_id = os.environ["GOOGLE_CLOUD_PROJECT"]
+    # construct a date object representing yesterday
+    yesterday = datetime.now(timezone.utc) - timedelta(days=1)
+    # Cloud Logging expects a timestamp in RFC3339 UTC "Zulu" format
+    # https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry
+    time_format = "%Y-%m-%dT%H:%M:%S.%f%z"
+    # build a filter that returns GKE Admin Activity audit Logs from
+    # the past 24 hours
+    # https://cloud.google.com/kubernetes-engine/docs/how-to/audit-logging
```

Comment:
maybe add a word or two about how to compose a filter (best practices): 
- strive to scope the filter to minimal time period
- use indexed fields

Reply:
Hello,
I'm getting the following error associated with the filter string.  Thanks for your help !
filter_str = (
    f'logName="projects/{project_id}/logs/cloudaudit.googleapis.com%2Factivity"'
    f'AND resource.type="k8s_cluster"'
    f'AND timestamp >= "{yesterday.strftime(time_format)}"'
    )

InvalidArgument: 400 Unparseable filter: syntax error at line 1, column 8, token '"projects/np-sec-cybersec-data-science/logs/cloudaudit.googleapis.com%2Factivity"';
syntax error at line 1, column 108, token '"k8s_cluster"'

Reply:
@Ecoware 
One thing that stands out to me is you're missing the space before then ANDs. If you merged those lines into one string, the ANDs would be merged with their preceeding lines

Also, are you sure you have all your quotes formatted as in the sample? From your error message, it seems like the k8s_cluster part isn't being parsed as part of the string. Make sure k8s_cluster is surrounded by double quotes, but the surrounding `AND resource.type="k8s_cluster"` part is surrounded by single quotes.

Try adding a `print(filter_str)` and let me know what the result is if you're still having trouble